app/src/main/res/values/strings.xml

```diff
@@ -436,6 +436,9 @@
 
   <!-- Short code search -->
   <string name="shortcodesearchresult_enter_patient_name_button">Enter Patient\'s name</string>
+
+  <!-- Simple video -->
   <string name="simple_video_watch_video">Watch Video</string>
   <string name="simple_video_how_to_use_the_simple_app">How to use the Simple app</string>
+  <string name="simple_video_duration">5:07 minutes</string>
```

Comment:
Should we split the duration from minutes? If not, this could make i18n complicated if the duration of the video changes?

Reply:
Good point!